### PR TITLE
[CFM-910] Fixing unified workflow + adding a test

### PR DESCRIPTION
**Pull request by ordavidil**
https://issuetracker.amplexor.com/jira/browse/CFM-910

### CODE REVIEW

File: project/profiles/capacity4more/behat/features/unified_workflow.feature [hunk 1]
```diff
@@ -0,0 +1,9 @@
+Feature: Unified Workflow
+  Test unified workflow
+
+  @api
+  Scenario: Create a comment on a task while changing some task fields and
```
Comment (soniCaH) [inline project/profiles/capacity4more/behat/features/unified_workflow.feature]: @ordavidil 
Seems like a strange title for this scenario

Comment (ordavidil) [inline project/profiles/capacity4more/behat/features/unified_workflow.feature]: @soniCaH 
haha,  damn,  nothing can come out perfect ...

ps, I've copied it from here:
https://github.com/capacity4dev/capacity4more/blob/develop/project/profiles/capacity4more/behat/features/installation.feature#L5

I was sure it is a default title for scenarios ;)

Comment (soniCaH) [inline project/profiles/capacity4more/behat/features/unified_workflow.feature]: Haha, I see. That doesn't make sense there either :-D

**Comment by ordavidil** [comment]
@soniCaH 
I've fixed the title skipping the ci test (not really needed).

**Pull request merged**